bpf/init.sh

```diff
@@ -277,6 +278,11 @@ function bpf_load_cgroups()
 		return 1
 	fi
 
+	set +e
```

Comment:
nit: Do we need `set +e` _and_ `|| true`?

Reply:
```
λ  ~  bpftool link detach pinned /foo
Error: bpf obj get (/foo): No such file or directory
λ  ~  echo $?
1
```
I think so, unless I'm missing something obvious.

Reply:
This is where I was coming from:

```
$ cat foo.sh
#!/bin/bash

set +e
bpftool link detach pinned /foo
echo "The script continued after the previous command returned $?"
set -e
$ ./foo.sh
Error: bpf obj get (/foo): Operation not permitted
The script continued after the previous command returned 1

Reply:
(either way, we don't really have a style guide for this and I see other areas adjusting `set +e` ...do something that may have errors... `set -e`, so :shrug: )

Reply:
init.sh has `set -e` at the top, so omitting `set +e` would be more akin to:
```
λ  ~  cat foo.sh
#!/bin/sh

set -e

bpftool link detach pinned /foo
echo "The script continued after the previous command returned $?"
λ  ~  sh foo.sh
Error: bpf obj get (/foo): No such file or directory
```

Reply:
Right, hence the question whether `set +e` is redundant with `|| true`. Put another way:

```
$ cat foo.sh 
#!/bin/bash

set -e

bpftool link detach pinned /foo || true
echo "The script continued because of the '|| true' (exit code $?)"
$ ./foo.sh 
Error: bpf obj get (/foo): Operation not permitted
The script continued because of the '|| true' (exit code 0)
```